.github/workflows/post-merge-push.yaml

```diff
@@ -0,0 +1,61 @@
+name: Build latest binaries
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-push:
+    concurrency:
+      group: build-${{ github.head_ref || github.run_id }}
+      cancel-in-progress: true
+    name: Build and Unit Test
+    env:
+      REGISTRY_REPO: oci://ghcr.io/k0rdent/kcm/charts-ci
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.vars.outputs.version }}
+    permissions:
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ''
```

Comment:
The checkout workflow will default to the code that triggered the action. This isn't needed and leaves some ambiguity if it is still checking out the default code or doing something else with an empty string.